plugins/node/instrumentation-undici/test/fetch.test.ts

```diff
@@ -57,6 +54,9 @@ describe('UndiciInstrumentation `fetch` tests', function () {
       this.skip();
     }
 
+    instrumentation = new UndiciInstrumentation();
```

Comment:
Note for reviewer: the test runner was loading all the test suites at once and also evaluating the 1st `describe` block. As a consequence there were 3 instances of the instrumentation before any test was ran. Leading to errors because all instances where getting messages from diagnostics channels

Reply:
Nice find!